.travis.yml

```diff
@@ -41,4 +41,14 @@ matrix:
           fi
         - docker build -t asegroup1/backend-$DOCKER_BRANCH:latest .
         - docker push asegroup1/backend-$DOCKER_BRANCH:latest
-
+    - language: java
```

Comment:
Would this not cause the java to be built twice? Or is that necessary for the sonar scan